dotcom-rendering/src/web/layouts/ImmersiveLayout.tsx

```diff
@@ -220,20 +265,201 @@ export const ImmersiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 
 	const mainMedia = CAPIArticle.mainMediaElements[0] as ImageBlockElement;
 	const captionText = decideCaption(mainMedia);
+	const HEADLINE_OFFSET = mainMedia ? 120 : 0;
 	const { branding } =
 		CAPIArticle.commercialProperties[CAPIArticle.editionId];
 
 	const contributionsServiceUrl = getContributionsServiceUrl(CAPIArticle);
 
+	const palette = decidePalette(format);
+
 	const isLabs = format.theme === ArticleSpecial.Labs;
 
-	return (
-		<>
-			<ImmersiveHeader
-				CAPIArticle={CAPIArticle}
-				NAV={NAV}
+	/**
+	now that the nav bar and labs header aren't included in the div which the
+	hasMainMediaStyles are applied to, we need change the height values
```

Comment:
great to have the inline documentation of why this is added!

can I ask where the pixel values `103` and `48` come from here though? I can't find reference in the prior code to either of these values, and I'm a bit concerned that these values might be fragile to future changes to the header? Either way, it would be good to document the reasons, if they aren't already documented elsewhere?

Reply:
Great point, these are the heights of the nav bar and labs header but it's not clear at all. I'll document this and see if I can import them to make the code less fragile

Reply:
This is great @emma-imber 🎉 

Minor: suggest losing the bit at the start (up until the comma) since I suspect nobody in the future needs to worry about the way things used to work.